minio/minio
# Fix the NSQ environment key

## Description

There was a typo in the ENV key declaration for NSQ.

The `mergeTargets` function was misbehaving as it considered `MINIO_NOTIFY_NSQ` as the prefix and returned all the ENVs substrings as targets.

## Motivation and Context
Fixes #10100

## How to test this PR?
Set the ENVs as described in the issue and start the server. You should not see any non-related errors.

## Types of changes

## Checklist:
